src/eslint-config-adeira/__tests__/__fixtures__/valid-eslint-examples/flow/exhaustive-checking.js

```diff
@@ -19,18 +19,3 @@ export function exhaustA(x: Cases) {
     (x: empty);
   }
 }
-
-export function exhaustB(
-  reason: 'magicLink' | 'signUpConfirmation' | 'resetPassword',
-): void | string {
-  switch (reason) {
-    case 'magicLink':
-      return __('account.check_email_magic_link');
-    case 'signUpConfirmation':
-      return __('account.check_email_sign_up');
-    case 'resetPassword':
-      return __('account.you_will_recieve_password');
-    default:
-      return invariantMock(false, 'Unsupported reason: %j', (reason: empty));
-  }
-}
```

Comment:
Can we leave this here modified? It's not about the `__` but rather about the invariant and reason used directly while checking whether the type is exhausted.